tools/dds/dds-server/lrs-device-controller.cpp

```diff
@@ -2,99 +2,205 @@
 // Copyright(c) 2022 Intel Corporation. All Rights Reserved.
 
 #include "lrs-device-controller.h"
+
 #include <librealsense2/utilities/easylogging/easyloggingpp.h>
+
+#include <realdds/dds-device-server.h>
+#include <realdds/dds-stream-server.h>
+
 #include <algorithm>
 #include <iostream>
 
+using nlohmann::json;
+using namespace realdds;
 using namespace tools;
 
-// This is a wrapper around a RealSense sensor, automatically opening and streaming it in the ctor
-// and stopping/closing on destruction
-class lrs_device_controller::lrs_sensor_streamer
+
+std::shared_ptr< dds_stream_profile > create_dds_stream_profile( rs2_stream type, nlohmann::json const & j )
```

Comment:
I'm not clear why we're creating new profiles rather than finding an existing profile that's part of the server streams?

Reply:
Can be an improvement on the tool. Not for this PR